replica-lib/src/accountsdb_repl_server.rs

```diff
@@ -0,0 +1,177 @@
+use {
+    futures_util::FutureExt,
+    log::*,
+    std::{
+        net::SocketAddr,
+        sync::{Arc, RwLock},
+        thread::{self, Builder, JoinHandle},
+    },
+    tokio::{
+        runtime::Runtime,
+        sync::oneshot::{self, Receiver, Sender},
+    },
+    tonic::{self, transport},
+};
+
+tonic::include_proto!("accountsdb_repl");
+
+pub trait ReplicaUpdatedSlotsServer {
```

Comment:
nit: I think "updated" -> "confirmation" might be a little clearer here, i.e.

`ReplicaUpdatedSlotsServer -> ReplicaSlotConfirmationServer`

Reply:
Yes. Will address

Reply:
Done